rebasehelper/plugins/build_tools/rpm/copr_.py

```diff
@@ -41,6 +41,21 @@ class Copr(BuildToolBase):
     Class representing Copr build tool.
     """
 
+    OPTIONS = [
+        {
+            'name': ['--copr-permanent-project'],
```

Comment:
Why not `--copr-project-permanent`?

Reply:
I meant permanent as an adjective tied to project. Though you are right, that `--copr-project-permanent` is better, it implies that bot of the added options influence the project.